charts/openshift-metering/templates/hadoop/hadoop-scripts.yaml

```diff
@@ -3,6 +3,14 @@ kind: ConfigMap
 metadata:
   name: hadoop-scripts
 data:
+  azure_secret_copy.sh: |
```

Comment:
This does more than azure stuff. Let's rename this to something like `copy-hadoop-config.sh`.